ansible_rulebook/vault.py

```diff
@@ -0,0 +1,82 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from ansible.parsing.dataloader import DataLoader
```

Comment:
The CI runs fine. Does it mean the dependency is already added?

Reply:
The CI runs fine because we are installing ansible in the pipeline and in the dockerfile to satisfy an optional dependency for run_playbook, etc...
But now is a required library of the project, so It should be present in the setup.cfg.